### PR TITLE
chore: 🤖 upgrade markuplint

### PR DESCRIPTION
**Pull request by kodai3**
to fix the workaround I introduced [here](https://github.com/ubie-oss/ubie-ui/pull/18#discussion_r1514759057), because it is fixed with [this](https://github.com/markuplint/markuplint/pull/1524). 🎉

This PR is to make sure it is fixed without degradation, I'll leave it up to you if you merge even alpha version. 👍

### CODE REVIEW

File: src/stories/Stack.stories.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const Nested: Story = {
 
 export const MarkupAsList: Story = {
   render: (args) => (
-    <Stack {...args} as="li">
+    <Stack {...args} as="ul">
```
Comment (kodai3) [inline src/stories/Stack.stories.tsx:63]: This is not related to the switch component issue, but it came up upgrading `markuplint` major version.

**Review by takanorip — APPROVED** [review]
🎉

**Comment by takanorip** [comment]
Leave this PR unmerged.
I will merge this PR when that modification is officially released. 👍

**Comment by kodai3** [comment]
@takanorip 
markuplint [v4.4.0](https://github.com/markuplint/markuplint/releases/tag/v4.4.0) is released. 🎉🎉
We can merge this now.

**Review by takanorip — APPROVED** [review]
LGTM!!!

**Pull request merged**